DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_static_entries.py

```diff
@@ -0,0 +1,136 @@
+import pytest
+import asyncio
+
+from dent_os_testbed.lib.bridge.bridge_fdb import BridgeFdb
+from dent_os_testbed.lib.bridge.bridge_link import BridgeLink
+from dent_os_testbed.lib.ip.ip_link import IpLink
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+    tgen_utils_get_traffic_stats,
+    tgen_utils_setup_streams,
+    tgen_utils_start_traffic,
+    tgen_utils_stop_protocols,
+    tgen_utils_stop_traffic,
+    tgen_utils_get_loss,
+    tgen_utils_dev_groups_from_config,
+    tgen_utils_traffic_generator_connect,
+)
+
+pytestmark = [pytest.mark.suite_functional_bridging, pytest.mark.asyncio]
+
+async def test_bridging_static_entries(testbed):
+    """
+    Test Name: test_bridging_static_entries
+    Test Suite: suite_functional_bridging
+    Test Overview: Verify static bridge entries.
+    Test Author: Kostiantyn Stavruk
+    Test Procedure:
+    1.  Init bridge entity br0.
+    2.  Set ports swp1, swp2, swp3, swp4 master br0.
+    3.  Set bridge br0 admin state UP.
+    4.  Set entities swp1, swp2, swp3, swp4 UP state.
+    5.  Set ports swp1, swp2, swp3, swp4 learning OFF.
+    6.  Set ports swp1, swp2, swp3, swp4 flood OFF.
+    7.  Adding FDB static entries for ports swp1, swp2, swp3, swp4.
+    8.  Send traffic with matching destination macs.
+    9.  Verify that address have been learned and forwarded.
+    """
+
+    bridge = "br0"
+    tgen_dev, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 4)
+    if not tgen_dev or not dent_devices:
+        print("The testbed does not have enough dent with tgen connections")
+        return
+    dent_dev = dent_devices[0]
+    device_host_name = dent_dev.host_name
+    tg_ports = tgen_dev.links_dict[device_host_name][0]
+    ports = tgen_dev.links_dict[device_host_name][1]
+    traffic_duration = 5
+
+    out = await IpLink.add(
```

Comment:
You can move your bridge setup to a fixture and use it on the module level. Anyway each test require it.